cartographer_ros/configuration_files/sim_chuck.lua

```diff
@@ -0,0 +1,77 @@
+-- Copyright 2016 The Cartographer Authors
```

Comment:
This file name isn't very descriptive. Can we change it to show what it's meant for?